sci-misc/boinc/files/boinc.init

```diff
@@ -54,6 +61,13 @@ env_check() {
 		return 1
 	fi
 
+	# The boinccmd is crucial, or we can not attach, suspend or resume
+	# the boinc client
+	if [ ! -x "$BOINCCMD" ] ; then
```

Comment:
seeing that you're already using bashisms, prefer `[[ ]]` over `[ ]` as double brackets are more robust in bash than crappy single POSIX brackets